galgebra/lt.py

```diff
@@ -486,28 +488,42 @@ def matrix(self):
-    j we have (a is a scalar and u_j a vector)
-          F(v_1,...,a*v_j,...,v_r) = a*F(v_1,...,v_j,...,v_r)
-          F(v_1,...,v_j+u_j,...,v_r) = F(v_1,...,v_j,...,v_r) + F(v_1,...,u_j,...,v_r).
-    If F and G are two mlt's with the same number of argument slots then the sum is
-          (F+G)F(v_1,...,v_r) = F(v_1,...,v_r) + G(v_1,...,v_r).
-    If F and G two mlt's with r and s argument slots then their product is
-          (F*G)(v_1,...,v_r,...,v_(r+s)) = F(v_1,...,v_r)*G(v_(r+1),...,v_(r+s)),
-    where * is any of the multivector multiplicative operations.
-    The derivative of a mlt with is defined as the directional derivative with respect
-    with respect to the coordinate vector (we assume F is implicitely a function of the
+    a list of vectors (*kargs) :math:`F(v_1,...,v_r)` where for any argument slot
+    :math:`j` we have (:math:`a` is a scalar and :math:`u_j` a vector)
+
+    .. math::
+          F(v_1,...,a*v_j,...,v_r) &= a*F(v_1,...,v_j,...,v_r) \\
+          F(v_1,...,v_j+u_j,...,v_r) &= F(v_1,...,v_j,...,v_r) + F(v_1,...,u_j,...,v_r).
+
+    If F and G are two `Mlt`\ s with the same number of argument slots then the sum is
+
+    .. math:: (F+G)F(v_1,...,v_r) = F(v_1,...,v_r) + G(v_1,...,v_r).
+
+    If :math:`F` and :math:`G` are two `Mlt`\ s with :math:`r` and :math:`s`
+    argument slots then their product is
+
+    .. math:: (F*G)(v_1,...,v_r,...,v_{r+s}) = F(v_1,...,v_r)*G(v_{r+1},...,v_{r+s}),
+
+    where :math:`*` is any of the multivector multiplicative operations.
+    The derivative of a `Mlt`\ with is defined as the directional derivative with respect
+    to the coordinate vector (we assume :math:`F` is implicitely a function of the
     coordinates)
-        F(v_1,...,v_r;v_(r+1)) = (v_(r+1)|grad)F(v_1,...,v_j,...,v_r).
-    The contraction of a mlt between slots j and k is defined as the
-    geometric derivative of F with respect to slot k and the inner geometric
-    derivative with respect to slot j (this gives the standard tensor
-    definition of contraction for the case that F is a scalar function)
-        Contract(i,j,F) = grad_i|(grad_j*F(v_1,...,v_i,...,v_j,...,v_r))
-                        = grad_j|(grad_i*F(v_1,...,v_i,...,v_j,...,v_r)).
-    This returns a mlt with slot i and j removed.
+
+    .. math:: F(v_1,...,v_r;v_{r+1}) = (v_{r+1}|\Nabla)F(v_1,...,v_j,...,v_r).
+
+    The contraction of a `Mlt`\ between slots :math:`j` and :math:`k` is defined as the
+    geometric derivative of :math:`F` with respect to slot :math:`k` and the inner geometric
+    derivative with respect to slot :math:`j` (this gives the standard tensor
+    definition of contraction for the case that :math:`F` is a scalar function)
+
+    .. math::
+
+        Contract(i,j,F) &= \Nabla_i|(\Nabla_j*F(v_1,...,v_i,...,v_j,...,v_r)) \\
+                        &= \Nabla_j|(\Nabla_i*F(v_1,...,v_i,...,v_j,...,v_r)).
+
+    This returns a `Mlt`\ with slot :math:`i` and :math:`j` removed.
```

Comment:
@abrombo: Is `\Nabla` a correct interpretation of what you meant by `grad` here? Or would `\mathtt{grad}` be better?

Reply:
Why isn't? It seems to be the right one to use in a formula instead of in code?

Reply:
Well it's sort of tied to the operator semantics, and both `|` and `*` seem a little out of place in this formula. Perpahsp they should be `\cdot` and ``, represpectively

Reply:
The original doc seems to be demonstrating code usage, thus the `|` and `*`. Was it you who changed them to Math formula?

Reply:
Yes, I did, because it makes the subscripts easier to read

Reply:
Then might just change to `\cdot` and `` as well.

Reply:
I use `\nabla` for `grad` in output but that is just an annotation choice when printing out `grad`.

Reply:
Updated to use `\bullet` and implicit multiplication, thanks for the input